setup.cfg

```diff
@@ -1,14 +1,16 @@
 [metadata]
 name = matrix_common
-description = "Common utilities for Synapse, Sydent and Sygnal"
+description = Common utilities for Synapse, Sydent and Sygnal
 long_description = file: README.md
 long_description_content_type = text/markdown
-version = 0.0.0
+version = 1.0.0
 
 classifiers =
   License :: OSI Approved :: Apache Software License
 
 [options]
+packages =
+  matrix_common
 python_requires = >= 3.7
```

Comment:
Synapse still supports 3.6, but only for a few more weeks. Not sure of your plan for when we intend to use this?

Reply:
Good point. I'm hoping to use this sooner rather than later, so we really ought to support 3.6 for now.